### PR TITLE
Azure docs

### PR DESCRIPTION
**Pull request by ArendPeter**
## Description

You can preview the new docs here: https://arendpeter.github.io/star-server/

Changes
* Organized the docs into folders
* Moved everything under a contributions to make space for other documentation
* Documented Azure onboarding
* Updated trouble shooting with logging instructions

### CODE REVIEW

File: docs/contributions/0_contribution_guide.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# STAR.VOTE contribution guide
+
+We're so excited you're interested in contributing!
+
+Start by joining our community [here](https://starvoting.us/join). Once you fill out the form you'll receive an invite to our slack where you can coordiate with the community
+
+You can also view our calendar [here](https://www.starvoting.org/calendar) and join our weekly software meetings. 
+ 
+Once you're ready to start contributing you can go to [the next step](contributions/setup_locally.html) for building your local setup
```
Comment (mikefranze) [inline docs/contributions/0_contribution_guide.md:16]: This link doesn't work, missing the "1_". A few other links aren't working as well.

File: docs/contributions/1_setup_locally.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
 layout: default
-title: Set up star.vote locally
-nav_order: 2000
+title: 🌱 First time Local Setup
+nav_order: 1
+parent: Contribution Guide
 ---
 
 # Set up star.vote locally
```
Comment (mikefranze) [inline docs/contributions/1_setup_locally.md:8]: I shut down our Heroku apps today, so we should switch to referencing dev.star.vote

Comment (ArendPeter) [reply docs/contributions/1_setup_locally.md]: Pushed a pass removing the heroku references, some of the older docs will probably need some more updates though for them to be compatible with the new setup

File: docs/contributions/0_contribution_guide.md (hunk 1 above)
Comment (ArendPeter) [reply docs/contributions/0_contribution_guide.md]: Fixed! (and did a pass fixing a few others as well)

**Review by mikefranze — APPROVED** [review]

**Pull request merged**